### PR TITLE
[FIX] Structure tool metrics error fix

### PR DESCRIPTION
**Pull request by Deepak-Kesavan**
## What

- Handled case where metrics is missing in the output

## Why

- Key error happens when metrics key is missing

## How

-

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

-

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: tools/structure/src/main.py
Comment (chandrasekharan-zipstack) [inline tools/structure/src/main.py:1]: @Deepak-Kesavan please bump the tool version as well

File: tools/structure/src/main.py [hunk 1]
```diff
@@ -253,7 +253,7 @@ def run(
             structured_output_dict[SettingsKeys.METADATA] = metadata
             structured_output = json.dumps(structured_output_dict)
 
-        metrics = structured_output_dict[SettingsKeys.METRICS]
+        metrics = structured_output_dict.get(SettingsKeys.METRICS, {})
```
Comment (chandrasekharan-zipstack) [inline tools/structure/src/main.py:256]: @Deepak-Kesavan since we always set `capture_metrics` to `True` -> why do you think this metrics key is missing?

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM - but please ensure we bump the tool version in this PR itself @Deepak-Kesavan

File: tools/structure/src/main.py (hunk 1 above)
Comment (Deepak-Kesavan) [reply tools/structure/src/main.py:256]: It was not handled for table and record type

Comment (harini-venkataraman) [inline tools/structure/src/main.py:256]: @Deepak-Kesavan Hope it is handled for `KeyError` in places where we use this variable when the dict is empty.

**Review by harini-venkataraman — APPROVED** [review]
Added a minor comment, LGTM otherwise.

File: tools/structure/src/main.py (hunk 1 above)
Comment (Deepak-Kesavan) [reply tools/structure/src/main.py:256]: @harini-venkataraman Yes it is handled.

File: tools/structure/src/main.py
Comment (chandrasekharan-zipstack) [reply tools/structure/src/main.py:1]: Necessary changes were made in the cloud patch and the image was updated. OSS tool needs no action here - hence ignoring the bump for this

**Pull request merged**